content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -125,7 +125,8 @@ different Kubernetes components.
 | `HugePageStorageMediumSize` | `true` | Beta | 1.19 | |
 | `IPv6DualStack` | `false` | Alpha | 1.15 | |
 | `ImmutableEphemeralVolumes` | `false` | Alpha | 1.18 | 1.18 |
-| `ImmutableEphemeralVolumes` | `true` | Beta | 1.19 | |
+| `ImmutableEphemeralVolumes` | `true` | Beta | 1.19 | 1.20 |
```

Comment:
Hi @wojtek-t .
Just noticed that this information should be in the "GA" table.
Would you move the three `ImmutableEphemeralVolumes` entries into the next table?

Reply:
done

Reply:
Great!
/lgtm